recipes/validators/meta.yaml

```diff
@@ -36,22 +36,23 @@ requirements:
   build:
     # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
     # Compilers are named 'c', 'cxx' and 'fortran'.
-    - {{ compiler('c') }}
+    # - {{ compiler('c') }}
   host:
     - python
     - pip
   run:
     - python
+    - decorator
```

Comment:
The Bioconda recipes has version constraints on the dependencies:
https://github.com/bioconda/bioconda-recipes/blob/453ba6844a273f113889d5d3f2a4fb17a1724b48/recipes/validators/meta.yaml#L24-L25
Those are from https://github.com/kvesteri/validators/blob/0.14.0/setup.py#L36=L37.
Please add them here.